### PR TITLE
Pull request for WAZO-3149-asterisk-debhelper

### PR DESCRIPTION
**Pull request by fblackburn1**
**Before Merge:**
- Depends-on: https://github.com/wazo-platform/asterisk-vanilla/pull/4
- Depends-on: https://github.com/wazo-platform/asterisk-debug/pull/5

### CODE REVIEW

File: debian/asterisk.prerm [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#! /bin/sh
+
+set -e
+
+rmdir /usr/local/share/asterisk/sounds 2>/dev/null || true
```
Comment (pc-m) [inline debian/asterisk.prerm:5]: You can ignore this comment if you want
`rm --ignore-fail-on-non-empty /usr/local/share/asterisk/sounds`

**Review by pc-m — COMMENTED** [review]
Only commenting at this point since I did not get a chance to test the packages yet.

good job bringing this package back from the past

File: debian/control [hunk 2]
```diff
@@ -5,51 +5,42 @@ Maintainer: Wazo Maintainers <dev@wazo.community>
...
+ libsnmp-dev,
  libspandsp-dev,
- libspeex-dev (>= 1.1.12-3),
+ libspeex-dev,
  libspeexdsp-dev,
- libsqlite-dev,
  libsqlite3-dev,
  libsrtp0-dev (>= 1.5.4),
- libss7-dev (>= 2.0),
+ libss7-dev,
```
Comment (sboily) [inline debian/control:34]: we don't need it.

File: debian/control [hunk 3]
```diff
@@ -5,51 +5,42 @@ Maintainer: Wazo Maintainers <dev@wazo.community>
...
  libogg-dev,
- libopenr2-dev,
  libpopt-dev,
  libpq-dev,
- libpri-dev (>= 1.4.7),
  libfreeradius-dev,
  libreadline-dev,
  libresample1-dev,
- libsnmp9-dev | libsnmp-dev,
+ libsnmp-dev,
```
Comment (sboily) [inline debian/control:28]: dont need it

**Review by sboily — CHANGES_REQUESTED** [review]

File: debian/asterisk.prerm (hunk 1 above)
Comment (sduthil) [reply debian/asterisk.prerm:5]: That was taken from the original Debian packaging: https://salsa.debian.org/pkg-voip-team/asterisk/-/blob/debian/latest/debian/asterisk.prerm. I think we'll avoid diverging from them if not necessary.

**Review by sduthil — APPROVED** [review]

**Pull request merged**